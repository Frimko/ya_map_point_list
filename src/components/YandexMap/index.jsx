import React from 'react';
import { YMaps, Map } from 'react-yandex-maps';
import Placemarks from 'containers/Placemarks';
import config from 'src/config';

const mapState = {
  center: config.mapCentrCoordinates,
  zoom: 10,
  // controls: []
};

class YandexMap extends React.PureComponent {
  render() {
    return (
      <YMaps>
        <Map
          state={mapState}
          width="100%"
          height="100%"
        >
          <Placemarks/>
        </Map>
      </YMaps>
    );
  }
}

export default YandexMap;
