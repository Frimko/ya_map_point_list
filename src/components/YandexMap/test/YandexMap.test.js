import React from 'react';
import { shallow } from 'enzyme';
import YandexMap from '../index';

describe('YandexMap component', () => {

  describe('snapshot', () => {
    test('snapshot', () => {
      expect(shallow(<YandexMap/>)).toMatchSnapshot();
    });
  });
});
