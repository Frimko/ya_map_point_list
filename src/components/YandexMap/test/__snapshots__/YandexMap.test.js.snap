// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`YandexMap component snapshot snapshot 1`] = `
<YMaps
  enterprise={false}
  onApiAvaliable={[Function]}
  version="2.1"
>
  <Map
    height="100%"
    instanceRef={[Function]}
    state={
      Object {
        "center": Array [
          55.76,
          37.64,
        ],
        "zoom": 10,
      }
    }
    width="100%"
  >
    <Connect(Placemarks) />
  </Map>
</YMaps>
`;
